functions/firestore/ItemHelper.js

```diff
@@ -15,9 +15,12 @@ const createItem = body => new Promise(async (resolve, reject) => {
 
 const modifyItem = body => new Promise(async (resolve, reject) => {
   try {
-    console.log(`Updating item '${body.itemId}'...`);
+    console.log(`Updating item '${(body.itemId)}'...`);
+    const oldState = (await REFS.COLLECTIONS.ITEMS.doc(body.itemId).get()).data();
     await REFS.COLLECTIONS.ITEMS.doc(body.itemId).update(body);
-    // TODO if still AVAIALABLE, call findMatchingItems
+    if (oldState.status !== ITEM_STATUSES.AVAILABLE && body.status === ITEM_STATUSES.AVAILABLE) {
```

Comment:
az elso check miert kell bele? ha elerheto itemet frissitek, akkor is nezzuk, hogy van-e match, nem?

Reply:
Annak csak akkor van ertelme, ha a tipus vagy a kategoria valtozhatna, de ugy remlik, hogy azokat nem lehet frissiteni.

Reply:
categoryId-t fogad optionalkent az updateItem, de vedd ki ha szerinted nem kellene

Reply:
Ez egy kerdest megerhet a kozosben.